spec/factories/hall.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :hall do
+    name { 'My name' }
+    capacity { 20 }
```

Comment:
```suggestion
    capacity { Faker::Number.number(digits: 2).to_i }
```

You can use Faker for all the factiories unless you need to have specific value.
I gave you example with casting numbers to integer.